packages/dds/map/src/test/mocha/directoryFuzzTests.spec.ts

```diff
@@ -3,344 +3,30 @@
-			clients.find((s) => s.containerRuntime.clientId === id) ?? {};
-		if (sharedDirectory !== undefined) {
-			console.log(`Client ${id}:`);
-			console.log(
-				JSON.stringify(sharedDirectory.getAttachSummary(true).summary, undefined, 4),
-			);
-			console.log("\n");
-		}
-	}
-}
-
-function makeReducer(loggingInfo?: LoggingInfo): AsyncReducer<Operation, FuzzTestState> {
-	const withLogging =
-		<T>(baseReducer: AsyncReducer<T, FuzzTestState>): AsyncReducer<T, FuzzTestState> =>
-		async (state, operation) => {
-			if (loggingInfo !== undefined && loggingInfo.printConsoleLogs) {
-				logCurrentState(state.clients, loggingInfo);
-				console.log("-".repeat(20));
-				console.log("Next operation:", JSON.stringify(operation, undefined, 4));
-			}
-			try {
-				await baseReducer(state, operation);
-			} catch (error) {
-				if (loggingInfo !== undefined) {
-					logCurrentState(state.clients, loggingInfo);
-				}
-				throw error;
-			}
-			return state;
-		};
-
-	const reducer: AsyncReducer<Operation, FuzzTestState> = combineReducersAsync({
-		createSubDirectory: async ({ client }, { path, name }) => {
-			const dir = client.channel.getWorkingDirectory(path);
-			assert(dir);
-			dir.createSubDirectory(name);
-		},
-		deleteSubDirectory: async ({ client }, { path, name }) => {
-			const dir = client.channel.getWorkingDirectory(path);
-			assert(dir);
-			dir.deleteSubDirectory(name);
-		},
-		set: async ({ client }, { path, key, value }) => {
-			const dir = client.channel.getWorkingDirectory(path);
-			assert(dir);
-			dir.set(key, value);
-		},
-		clear: async ({ client }, { path }) => {
-			const dir = client.channel.getWorkingDirectory(path);
-			assert(dir);
-			dir.clear();
-		},
-		delete: async ({ client }, { path, key }) => {
-			const dir = client.channel.getWorkingDirectory(path);
-			assert(dir);
-			dir.delete(key);
-		},
-	});
-
-	return withLogging(reducer);
-}
+import {
+	baseDirModel,
+	dirDefaultOptions,
+	makeDirOperationGenerator,
+	makeDirReducer,
+	type DirOperation,
+} from "./fuzzUtils.js";
 
 describe("SharedDirectory fuzz Create/Delete concentrated", () => {
-	const options: OperationGenerationConfig = {
-		setKeyWeight: 0,
-		clearKeysWeight: 0,
-		deleteKeyWeight: 0,
-		createSubDirWeight: 2,
-		deleteSubDirWeight: 2,
-		maxSubDirectoryChild: 2,
-		subDirectoryNamePool: ["dir1", "dir2"],
-		validateInterval: defaultOptions.validateInterval,
-	};
-	const model: DDSFuzzModel<DirectoryFactory, Operation> = {
-		workloadName: "default directory 1",
-		generatorFactory: () => takeAsync(100, makeOperationGenerator(options)),
-		reducer: makeReducer({ clientIds: ["A", "B", "C"], printConsoleLogs: false }),
-		validateConsistency: async (a, b) => assertEquivalentDirectories(a.channel, b.channel),
-		factory: new DirectoryFactory(),
-	};
-
-	createDDSFuzzSuite(model, {
-		validationStrategy: { type: "fixedInterval", interval: defaultOptions.validateInterval },
+	createDDSFuzzSuite(baseDirModel, {
```

Comment:
this test suite had different options passed in which should be preserved as it looks intentional and reasonable (subdirectory create/delete focus)